ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less

```diff
@@ -230,6 +231,7 @@ link-primary {
 }
 
 .logout {
+	font-family: Arial, Helvetica, sans-serif;
```

Comment:
@rakeshkumar-verma : we are using the font family with san-serif as priority in button.css . why have we changed priority here ?

Reply:
Updated back to san-serif

Reply:
@rishabh-behl   updated font family as  sans-serif;